# Generated for NaoQi method ALBluetooth.disconnect
# Disconnect all bluetooth devices

---


